Customizing/global/plugins/Services/Repository/RepositoryObject/TalentAssessment/classes/Observations/ilObservationsOverviewTableGUI.php

```diff
@@ -37,7 +37,6 @@ public function __construct($a_parent_obj, array $values, array $observator, $a_
 
 	public function fillRow($row) {
 		$this->tpl->setVariable("TITLE", $row["title"]);
-
```

Comment:
Bitte in Zukunft darauf achten, dass die PRs und Commits nur signifikaten Änderungen enthalten.